.travis.yml

```diff
@@ -1,3 +1,31 @@
+dist: bionic
+
+branches:
+  only:
+    - master
+    - /^v(0|[1-9]\d*)\.(0|[1-9]\d*)-dev$/
+    - /^v(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$/
+
+services:
+  - docker
+
+addons:
+  apt:
+    packages:
+      - docker-ce
+
+before_install:
+  - mkdir -p ~/.local/bin
+  - curl -fsSL -o ~/.local/bin/dash-network https://raw.github.com/dashpay/dash-network-deploy/master/bin/dash-network && chmod +x ~/.local/bin/dash-network
+  - git clone https://$NETWORK_CONFIGS_GITHUB_USERNAME:$NETWORK_CONFIGS_GITHUB_TOKEN@github.com/nmarley/dash-network-configs ~/dash-network-configs
+  - cp ~/dash-network-configs/$NETWORK_CONFIGS_PATH/.env.example ~/dash-network-configs/$NETWORK_CONFIGS_PATH/.env
+
+script:
+  - cd ~/dash-network-configs/$NETWORK_CONFIGS_PATH
+  - ~/.local/bin/dash-network test $DEVNET
+  - cd $TRAVIS_BUILD_DIR
+  - docker run dashpay/platform-test-suite $DAPI_SEED --faucet-key=$FAUCET_PRIVATE_KEY
```

Comment:
Does it work already?

Reply:
Kind of

https://travis-ci.com/github/dashevo/evonet-test-ci

Reply:
It seems not https://travis-ci.com/github/dashevo/evonet-test-ci/jobs/328011587#L2299

Probably, we need to build the image before we merge it. Also, let's set the docker tag with an env variable in the travis build settings.